src/test/java/core/basesyntax/service/impl/FormatTransformerServiceImplTest.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertIterableEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class FormatTransformerServiceImplTest {
+    private static FormatTransformerServiceImpl transformerService;
+
+    @BeforeAll
+    static void setUp() {
+        transformerService = new FormatTransformerServiceImpl();
+    }
+
+    @Test
+    void formatData_validData_ok() {
```

Comment:
let's add more test cases here as currently you are only validating one positive scenario